core/templates/dev/head/pages/exploration-player-page/learner-experience/conversation-skin.directive.ts

```diff
@@ -823,6 +830,16 @@ oppia.directive('conversationSkin', [
                 return;
               }
             }
+
+            if (ENABLE_SOLICIT_ANSWER_DETAILS_FEATURE) {
+              LearnerAnswerInfoService.askLearnerForAnswerInfo(
+                answer, interactionRulesService);
+
+              if (LearnerAnswerInfoService.cAskLearnerForAnswerInfo()) {
```

Comment:
cAskLearnerForAnswerInfo, is this by mistake?

Reply:
removed